configure.ac

```diff
@@ -6,20 +6,20 @@ dnl # Run ./autogen.sh to build configure script
 dnl #
 dnl ##########################################################################
 
-AC_PREREQ(2.59)
+AC_PREREQ(2.63)
```

Comment:
Uhm no. :-)

Read it as "has been tested", not as "being continuously tested"! If we want to be sure we could bump it even higher, but it works on 2.63 right now and this stuff changes very rarely.

I think I'll do what you suggested, leave as is and edit the commit message. I'll be submitting the same patch to all other repos as well, to keep them in sync.